Pages/Incoming.cshtml.cs

```diff
@@ -105,22 +105,25 @@ private async Task<(int?, DateTimeOffset)> GetCommitInfo(GitHubInfo gitHubInfo,
             else
             {
                 var comparison = await GetCommitsBehindAsync(gitHubInfo, build);
-                foreach (var commit in comparison.Commits)
+                if (comparison != null)
                 {
-                    if (commit.Commit.Committer.Date < commitAge)
+                    foreach (var commit in comparison.Commits)
                     {
-                        commitAge = commit.Commit.Committer.Date;
+                        if (commit.Commit.Committer.Date < commitAge)
+                        {
+                            commitAge = commit.Commit.Committer.Date;
+                        }
                     }
-                }
 
-                if (comparison.Commits.Count == 0)
-                {
-                    commitAge = build.DateProduced;
-                }
+                    if (comparison.Commits.Count == 0)
+                    {
+                        commitAge = build.DateProduced;

```

Comment:
Maybe we should set the initial `commitAge` to `build.DateProduced` instead of `DateTimeOffset.Now`. Then we could get rid of this section and above only check for `if (gitHubInfo != null)`

Reply:
Like the updated version?